src/main.cpp

```diff
@@ -40,7 +40,11 @@
 
 #include "binding.h"
 
+#ifdef __WINDOWS__
+#include "resource.h"
+#elseif __LINUX__
```

Comment:
Yeah, I was not entirely sure, so did a quick google search and found some entries. Therefore I assumed GCC supports it.